dspace-api/src/main/java/org/dspace/core/Context.java

```diff
@@ -595,4 +596,8 @@ protected void finalize() throws Throwable
     public void shutDownDatabase() throws SQLException {
         dbConnection.shutdown();
     }
+    
+    public void clearCache() throws SQLException {
+    	((Session)this.getDBConnection().getSession()).clear();
```

Comment:
I think we should extend the DBConnection interface to have a clearCache() & have the hibernate implementation call this clear.
Having it in here ties the context object to close to the hibernate implementation (which I want to avoid if we ever want to drop hibernate).